client/src/main/resources/templates/RuntimeReporter/index.html

```diff
@@ -22,8 +22,20 @@
         <link rel="stylesheet" href="../css/bootstrap-table.min.css">
         <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
         <!--[if lt IE 9]><script src="../bower_components/html5shiv/dist/html5shiv.js"></script><script src="../bower_components/respond/dest/respond.min.js"></script><![endif]-->
-        <style type="text/css">.modal-dialog {width: 95%; min-height: 95%; padding: 0;}.modal-content {min-height: 100%; }
-        </style>
+        <style type="text/css">.modal-dialog {width: 95%; min-height: 95%; padding: 0;}.modal-content {min-height: 100%; } </style>
```

Comment:
Note this code is here temporary until I fix the next bug in this sprint.  ]
Issue is when clicking the page source button or image, no model dialog comes up because webpage complains the js functions aren't defined.
Main.js isn't being read in correctly which has these 2 js functions defined in it.  I will work on that and fix that in another PR.